easybuild/easyconfigs/m/MoloVol/MoloVol-1.1.1-foss-2023a.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'MoloVol'
+version = '1.1.1'
+
+homepage = 'https://molovol.com/'
+description = """
+MoloVol is a free, cross-plattform, scientific software for volume and surface computations of
+single molecules and crystallographic unit cells. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

i think this can be moved to GCC
```suggestion
toolchain = {'name': 'GCC', 'version': '2023a'}
```